### PR TITLE
fix: Last name shows null when left blanked on sign up

### LINKED ISSUE #1965: When sigining up, last name is optional and when left null , name is shown as "FIRSTNAME null" instead of "FIRSTNAME"
**Describe the bug**
When sigining up, last name is optional and when left null , name is shown as "FIRSTNAME null" instead of "FIRSTNAME"

**To Reproduce**
1. Sign up and leave the last name null.
2. Go to navigation drawer.
3. Click on 'accounts'.
4. Check the name on the top of accounts screen.

**Expected behavior**
Name should appear as "FIRSNAME" but it appears aws "FIRSTNAME null".

**GIF**

NewGIF

**Smartphone Info:**
Device - Xiomi Redmi Note 5
Android Version - Oreo 8.1

**Would you like to work on the issue?**
- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by njnaman**
Fixes #1965

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: last name is shown only when it is not null .

Screenshots for the change: 
Screenshot_2019-11-10-10-10-09-062_com eventyay organizer

### CODE REVIEW

**Review by rob729 — COMMENTED** [review]
Circle CI build failed. Check for errors

**Comment by liveHarshit** [comment]
> Circle CI build failed. Check for errors

Ignore Circle CI for now, just check Travis.

**Review by liveHarshit — DISMISSED** [review]

**Review by codedsun — DISMISSED** [review]

**Review by ShridharGoel — DISMISSED** [review]

File: app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailFragment.java [hunk 1]
```diff
@@ -198,8 +198,10 @@ public void onResume() {
 
             ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
 
-            if (firstName != null)
+            if (firstName != null && lastName!=null)
                 actionBar.setTitle(firstName + " " + lastName);
+            else if(firstName!=null)
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailFragment.java]:
```suggestion
            else if (firstName!=null)
```

Comment (njnaman) [reply app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailFragment.java]: done

**Review by liveHarshit — APPROVED** [review]

**Pull request merged**